### PR TITLE
Update to otel 0.14

### PR DESCRIPTION
**Pull request by adlerhurst**
Hi guys

As i was testing otel 0.14 i updated your project. Maybe the changes work with you.

Greets from Switzerland

### CODE REVIEW

File: exporter/metric/metric.go [hunk 1]
```diff
@@ -21,8 +21,9 @@ import (
 	"strings"
 	"time"
 
-	"go.opentelemetry.io/otel/api/global"
-	apimetric "go.opentelemetry.io/otel/api/metric"
+	global "go.opentelemetry.io/otel"
```
Comment (dashpole) [inline exporter/metric/metric.go:24]: No need to rename.  Lets stick with otel.

File: exporter/trace/trace_proto.go [hunk 2]
```diff
@@ -27,8 +27,8 @@ import (
 	tracepb "google.golang.org/genproto/googleapis/devtools/cloudtrace/v2"
 	statuspb "google.golang.org/genproto/googleapis/rpc/status"
 
+	opentelemetry "go.opentelemetry.io/otel"
```
Comment (dashpole) [inline exporter/trace/trace_proto.go:30]: can we change to use "otel"?

File: exporter/metric/error.go [hunk 3]
```diff
@@ -20,7 +20,8 @@ import (
 
 	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
 
-	apimetric "go.opentelemetry.io/otel/api/metric"
+	apimetric "go.opentelemetry.io/otel/metric"
```
Comment (dashpole) [inline exporter/metric/error.go:23]: to be consistent, can we switch to using `metric` instead of `apimetric`?

File: exporter/metric/metric.go [hunk 4]
```diff
@@ -21,8 +21,9 @@ import (
...
+	apimetric "go.opentelemetry.io/otel/metric"
```
Comment (dashpole) [inline exporter/metric/metric.go:25]: It would also be nice to use `metric`, rather than `apimetric` here.

**Review by dashpole — COMMENTED** [review]
Thanks for the help!  Just a few nits.

File: exporter/metric/metric.go [hunk 5]
```diff
@@ -21,8 +21,10 @@ import (
 	"strings"
 	"time"
 
-	global "go.opentelemetry.io/otel"
-	apimetric "go.opentelemetry.io/otel/metric"
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
```
Comment (dashpole) [inline exporter/metric/metric.go:24]: please keep non-opentelemetry imports grouped at the bottom.  This also includes the protobuf import.

**Comment by adlerhurst** [comment]
hi @dashpole 

thanks for your review. i've corrected the files.

I've also changed the used otel version in.
https://github.com/adlerhurst/opentelemetry-operations-go/blob/20c83ede56cac1b365904bda780e79bba9c9cf46/exporter/trace/version.go#L20
and
https://github.com/adlerhurst/opentelemetry-operations-go/blob/20c83ede56cac1b365904bda780e79bba9c9cf46/exporter/metric/version.go#L20

File: exporter/metric/metric.go (hunk 5 above)
Comment (adlerhurst) [reply exporter/metric/metric.go]: changed it in all files

**Review by dashpole — APPROVED** [review]

**Comment by dashpole** [comment]
closed/reopened to poke tests

**Pull request merged**